src/nnet3bin/Makefile

```diff
@@ -10,7 +10,7 @@ BINFILES = nnet3-init nnet3-info nnet3-get-egs nnet3-copy-egs nnet3-subset-egs \
    nnet3-shuffle-egs nnet3-acc-lda-stats nnet3-merge-egs \
    nnet3-compute-from-egs nnet3-train nnet3-am-init nnet3-am-train-transitions \
    nnet3-am-adjust-priors nnet3-am-copy nnet3-compute-prob \
-   nnet3-average nnet3-am-info nnet3-combine nnet3-latgen-faster
+   nnet3-average nnet3-am-info nnet3-combine nnet3-latgen-faster nnet3-copy
```

Comment:
We try to keep lines in the Makefile to 80 characters, you might want a line break here (with \)